src/cephadm/cephadm

```diff
@@ -3305,11 +3344,15 @@ def command_bootstrap():
         ])
 
     if mon_network:
-        logger.info('Setting mon public_network...')
+        logger.info(f"Setting mon public_network to {mon_network}")
         cli(['config', 'set', 'mon', 'public_network', mon_network])
+    
+    if cluster_network:
+        logger.info(f"Setting 'global' cluster_network to {cluster_network}")
+        cli(['config', 'set', 'global', 'cluster_network', cluster_network])
 
-    if ipv6:
-        logger.info('Enabling IPv6 (ms_bind_ipv6)')
+    if ipv6 or ipv6_cluster_network:
+        logger.info('Enabling IPv6 (ms_bind_ipv6) binding')
         cli(['config', 'set', 'global', 'ms_bind_ipv6', 'true'])
```

Comment:
I'm not sure if this is right.

I you have the public network configured with IPv4 and the cluster network with IPv6 then you're telling to all daemons to bind with IPv6 whereas you should do that only for the OSD nodes (`global` vs `osd`), right ?

Reply:
Makes sense. I guess the complication is the current functionality. currently if the mon ip is ipv6 the ms_bind_ipv6 is set at a global level, which allows any daemon to use ipv6 if that's what it's defined with right? I was trying to enable ipv6 binding if either the public network or cluster network had ipv6 definitions - i.e. if ipv6 is on the cluster, allow it for any service that wants to use it (global). Also, since ipv4 is not disabled the use of ipv6 addressing is not mandated